spec/valkyrie/indexers/access_controls_indexer_spec.rb

```diff
@@ -66,6 +66,7 @@ def configure(_ = nil)
       end
       after do
         Hydra.remove_possible_method(:configure)
+        Object.send(:remove_const, :Hydra)
```

Comment:
@tpendragon I do not think you need to Hydra.remove_possible if you are going to remove Hydra.